builddefs/common_features.mk

```diff
@@ -811,7 +811,7 @@ ifeq ($(strip $(PS2_MOUSE_ENABLE)), yes)
     PS2_ENABLE := yes
     SRC += ps2_mouse.c
     OPT_DEFS += -DPS2_MOUSE_ENABLE
-    OPT_DEFS += -DMOUSE_ENABLE
+    MOUSE_ENABLE := yes
```

Comment:
Pedantic, but can we move this up so it's next to the other enable line.

Reply:
yeah, not a problem.